### DISCUSSION TITLE
Navigation speak street names

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#136] [Q&A] [answered]
Provide more options for speak street names in navigation:
- **Normal**
- **Advanced**: street names are only spoken in the "Last announcement" before the turn
- **Off**

Available in "Settings | Voice guidance | Advanced | Street names" or search settings for "street".

<img src="https://user-images.githubusercontent.com/3484020/234188122-8b7d3499-a720-4d51-850b-bef14ae3e8c2.jpg" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
Implemented in [Cruiser 3.0.18](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
will test it on the next tour.

**Reply by devemux86** [reply]
The street name of the next turn is usually useful when approaching the next turn,
so an option to not fill all instructions with the street name might be more pleasing.

**Reply by HannesHoBu** [reply]
Yesterday I checked Cruiser for the first time and I was really pleased with the new advanced announcement setting. It reduces the information from my perspective to an optimum.

**Reply by devemux86** [reply]
@HannesHoBu thanks for checking Cruiser!

Cruiser offers many configuration options, so that users can customize the application to their needs:
interface color & size options, voice navigation distance settings, customizable auto zoom, ...

More new features are already in development. Please stay tuned! :slightly_smiling_face: 

If you have questions or ideas for improvements and new features, please let me know.
